miden/src/repl/mod.rs

```diff
@@ -330,3 +345,15 @@ fn print_mem_address(addr: u64, mem: &Word) {
     let mem_int = mem.iter().map(|&x| x.as_int()).collect::<Vec<_>>();
     println!("{} {:?}", addr, mem_int)
 }
+
+fn use_module() {
+    println!("Modules available for importing:");
+    println!("");
+    println!("std::math::u256");
+    println!("std::math::u64");
+    println!("std::math::secp256k1");
+    println!("");
+    println!("How it works:");
+    println!("!use std::math::u64");
+}
```

Comment:
nit: I'd rename the function into `print_stdlib_modules()`.

Also, instead of hard-coding modules, we should probably just iterate over modules in `stdlib` and print them out dynamically.